### PR TITLE
hOiiiii! 

### PR DESCRIPTION
**Pull request by Dobby233Liu**
(not any comments)

### CODE REVIEW

**Comment by TemTemmie** [comment]
hello

File: .middle box.js [hunk 1]
```diff
@@ -34,5 +34,7 @@ client.Dispatcher.on(Events.GATEWAY_READY, 0=> {
 		e.message.channel.send('Hello');
 		if (e.message.content == 'octo.noprog'){
 		e.message.channel.('I wasnt been programmed for that.');
-    	 (e.message.content == 'octo.hOi'){
+    			if (e.message.content == 'octo.no'){
+		e.message.channel.send('oh');
+			if (e.message.content == 'octo.hOi'){
```
Comment (Dobby233Liu) [inline .middle box.js]: I not have any Cats.

**Comment by Dobby233Liu** [comment]
Now have 1 Commits
I'm waiting Teach Comments

**Comment by TemTemmie** [comment]
So i rename node.js to .middle box.js ?

**Comment by Dobby233Liu** [comment]
Nooooo

**Comment by TemTemmie** [comment]
oh

**Pull request closed without merging**